src/guides/v2.3/javascript-dev-guide/widgets/widget_modal.md

```diff
@@ -218,6 +219,14 @@ The class that is assigned to parent when the modal is opened.
 
 **Default value**: `_has-modal`
 
+### `outerClickHandler` {#modal_outerClickHandler}
+
+The function that handles the click on the modal overlay. If `outerClickHandler` option is not specified, the [closeModal()](#modal_close) method is used for handling the click on the modal overlay.
```

Comment:
I'm not sure it's a great idea to use "click on the modal overlay" statement. The "modal overlay" definition might be ambiguous in this case. If you mean the area out of the modal window, it's not 100% overlay according to the classic definition. 
"overlay" usually means something that overlaps the background (so, the modal window itself in our case).